fix: account view priority and table size

cmunns opened the pull request:
- tightened up spacing on both mobile and desktop for table
- removed ctas
- added alternating bg and normalized icon size and positions for tooltips
- bumped up checkbox border for better contrast
- aligned and normalized grid system to align text along same vertical axis.
<img width="863" alt="Screen Shot 2022-06-08 at 1 47 04 PM" src="https://user-images.githubusercontent.com/1189523/172714348-aa9207e8-b307-42af-b3bc-9f1cc91d86cf.png">
<img width="1110" alt="Screen Shot 2022-06-08 at 11 46 19 AM" src="https://user-images.githubusercontent.com/1189523/172714349-7ffc1b7d-470b-4d64-aa96-0b654487a25c.png">

cmunns commented:
I also imported some new icons to normalize the size and stroke. The existing ones were a little wonky in a few places
<img width="316" alt="Screen Shot 2022-06-09 at 9 06 55 AM" src="https://user-images.githubusercontent.com/1189523/172893984-0eb01145-97da-433f-b77d-0ada3c3553af.png">

Looks like this is causing the build to fail because package-lock is being ignored and not matching.

jsdevel commented on packages/website/components/account/filesManager/fileRowItem.js:
```diff
@@ -1,12 +1,14 @@
-import PencilIcon from 'assets/icons/pencil';
-import { addTextToClipboard, truncateString } from 'lib/utils';
+import CheckIcon from '../../../assets/icons/check';
+// import InfoBIcon from '../../../assets/icons/infoB';
```
can you remove this comment?

jsdevel commented on packages/website/components/account/filesManager/fileRowItem.scss:
```diff
@@ -67,25 +69,33 @@ $file-size: auto;
           color: $pictonBlue;
         }
         .Tooltip {
```
can you delete this rule?

jsdevel commented on packages/website/components/account/filesManager/filesManager.scss:
```diff
@@ -36,13 +36,13 @@
 
   // Title
   > :first-child {
```
can you delete this?

jsdevel commented on packages/website/components/account/filesManager/filesManager.scss:
```diff
@@ -36,13 +36,13 @@
   .files-manager-title {
     display: flex;
     flex-direction: row;
     &:not(.has-upload-button) {
```
can you delete this?

jsdevel commented on packages/website/components/account/filesManager/filesManager.scss:
```diff
@@ -73,7 +73,7 @@
     top: 0.125rem;
     cursor: pointer;
-    margin-right: 3.125rem;
+    // margin-right: 3.125rem;
```
delete

jsdevel commented on packages/website/components/account/storageManager/storageManager.scss:
```diff
@@ -4,17 +4,14 @@
   display: flex;
-  padding-left: 0.6875rem;
-  padding-right: 3.9375rem;
+  // padding-left: 0.4rem;
```
delete

jsdevel commented on packages/website/modules/zero/components/tooltip/tooltip.js:
```diff
@@ -1,5 +1,8 @@
-import InfoAIcon from 'assets/icons/infoA';
+import { BsFillQuestionCircleFill as InfoAIcon } from 'react-icons/bs';
+
+// import InfoAIcon from 'assets/icons/infoA';
```
delete

olizilla reviewed (changes requested):
Nice! This is going in the right direction

- [x] fix dem conflicts 
- [x] convert the `?` icon to use the `i`. Let's pick one and use it in both places.
- [x] Improve the date format. Avoid wrapping, and remove the comma. If we have to, truncate the date and add the full timestamp as a title attr. The full date portion should always be visible.
- [x] Make the name column the most important thing. Consider moving the date to the right hand side next to size.
- [ ] if not too filddly, rework the text styles in the tooltip to make them look more coherent with the table layout. e.g. the occasionally long list of Storage Providers ID addresses could have a smaller font. As they are ID addresses they could shown in monospace font similar to CIDs, but have a play and see what feels right there.

cmunns commented:
The latest from main included some tabs to separate out views between uploaded via web and pinned via api. I don't see a reason to show the tabs at all if the user is only ever interacting with the gui. I also added more distinct "selected" states to the whole row and card (pictured) and removed the scale onHover to the table buttons. I want to be sure that someone can test this that has API based pins so that the tabs show up when they need to before we merge.
<img width="607" alt="Screen Shot 2022-06-13 at 12 52 40 PM" src="https://user-images.githubusercontent.com/1189523/173461039-df1cf790-b25d-4d26-bf6c-72c5ffe1a155.png">
<img width="1190" alt="Screen Shot 2022-06-13 at 3 04 56 PM" src="https://user-images.githubusercontent.com/1189523/173461032-e17c552f-99e4-4fd3-9bcc-428c4420dab2.png">

JeffLowe commented:
Pinging @olizilla & @jsdevel for rereview.

olizilla commented:
If you can right align the text in the size column when on wider screens / table layout, then you have a deal.

> The latest from main included some tabs to separate out views between uploaded via web and pinned via api. I don't see a reason to show the tabs at all if the user is only ever interacting with the gui.

this sounds significant. I've not been following the "show pinned uploads seperately" thread. We should find out who was building that and avoid undoing there changes with this PR without some checking in first.

JeffLowe commented:
@cmunns looks like if you can address Oli's remaining concern, we are good to go on this one:
"...right align the text in the size column when on wider screens / table layout"
